Dockerfile

```diff
@@ -0,0 +1,28 @@
+# CodeClimate specification: https://github.com/codeclimate/spec/blob/master/SPEC.md
+#
+# Build and run via:
+#   docker build -t codeclimate/codeclimate-reek . && docker run codeclimate/codeclimate-reek
+
+FROM codeclimate/alpine-ruby:b38
```

Comment:
>> although I wonder whether it shouldn’t be codeclimate/alpine-ruby:latest anyway)

Actually it should just be 

>> codeclimate/alpine-ruby

since docker will then automatically look for the tag "latest". 
Unfortunately this doesn't exist which feels a little weird indeed.
@mrb an insights from you regarding this?